### PR TITLE
add CORS info to viyaFonts doc

### PR DESCRIPTION
**Pull request by eheckel**
Added a note to the viyaFonts.css doc about needing CORS enabled. 

Also fixed two typos that were bothering me :)

### CODE REVIEW

File: documentation/docs/guides/export-report-package.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ When you use the SAS Visual Analytics SDK to connect directly to SAS Viya, the f
 
 Within the exported report package, a file named `viyafonts.css` references fonts from your SAS Viya deployment for use in your page. Including a link to this stylesheet in your HTML page enables it to access the fonts that were used to design the report.
 
-To retrieve these fonts, a live connection to SAS Viya is required. The fonts cannot be retrieved while offline.
+To retrieve these fonts, a live connection to a SAS Viya server with CORS enabled is required. The fonts cannot be retrieved while offline. For more information on enabling CORS, see [`Enable Cross-Origin Resource Sharing`](viya-setup#enable-cross-origin-resource-sharing).
```
[the remark this answers is not in the document]
Comment (joylashley) [reply documentation/docs/guides/export-report-package.md:38]: @brmorr, your suggestions look good to me. @eheckel, thanks for fixing the typos.

[the remark this answers is not in the document]
Comment (brmorr) [reply documentation/docs/guides/export-report-package.md:38]: Well, I pointed out the problems with what is there, but didn't really offer a suggestion.  Maybe something like

> To use these fonts the SAS Viya deployment must be accessible and have CORS enabled. For more information on enabling CORS, see [`Enable Cross-Origin Resource Sharing`](viya-setup#enable-cross-origin-resource-sharing).

[the remark this answers is not in the document]
Comment (eheckel) [reply documentation/docs/guides/export-report-package.md]: Thanks Brad, I applied the suggestion

[the remark this answers is not in the document]
Comment (joylashley) [reply documentation/docs/guides/export-report-package.md]: @eheckel, can you still make a minor change to Brad's suggestion? If so, add a comma after the "To use these fonts" phrase before the mention of SAS Viya.

File: documentation/docs/guides/export-report-package.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ When you use the SAS Visual Analytics SDK to connect directly to SAS Viya, the f
 
 Within the exported report package, a file named `viyafonts.css` references fonts from your SAS Viya deployment for use in your page. Including a link to this stylesheet in your HTML page enables it to access the fonts that were used to design the report.
 
-To retrieve these fonts, a live connection to SAS Viya is required. The fonts cannot be retrieved while offline.
+To use these fonts, the SAS Viya deployment must be accessible and have CORS enabled. For more information on enabling CORS, see [`Enable Cross-Origin Resource Sharing`](viya-setup#enable-cross-origin-resource-sharing).
```
Comment (eheckel) [inline documentation/docs/guides/export-report-package.md:38]: @joylashley here's the updated version! Added the comma here

**Pull request merged**